frontend/src/components/floating-menus/NodeCatalog.svelte

```diff
@@ -118,7 +143,13 @@
 					<TextLabel>{nodeCategory[0]}</TextLabel>
 				</summary>
 				{#each nodeCategory[1].nodes as nodeType}
-					<TextButton {disabled} label={nodeType.name} tooltip={$nodeGraph.nodeDescriptions.get(nodeType.name)} action={() => dispatch("selectNodeType", nodeType.name)} />
+					<TextButton
+						{disabled}
+						label={nodeType.name}
+						tooltip={$nodeGraph.nodeDescriptions.get(nodeType.name)}
+						emphasized={nodeType == flatNodeList[focusedNodeIndex]}
```

Comment:
<img width="253" height="327" alt="Image" src="https://github.com/user-attachments/assets/eb1b3b0d-2157-4255-8497-048a35e90978" />

I think emph is not ideal could you use the color used for mouse hover.